subprojects/docs/src/docs/userguide/migration/upgrading_version_7.adoc

```diff
@@ -145,40 +145,16 @@ The last set of classes have no external or internal usages and therefore were d
 - `NoOpChangeListener`
 - `DeferredUtil`
 
-[[exec_properties]]
-=== Properties deprecated in Exec task
-
-* The link:{groovyDslPath}/org.gradle.api.tasks.Exec.html#org.gradle.api.tasks.Exec:execResult[Exec.execResult]
-property has been deprecated.
-Use the link:{groovyDslPath}/org.gradle.api.tasks.Exec.html#org.gradle.api.tasks.Exec:executionResult[Exec.executionResult]
-property instead.
-
 [[java_exec_properties]]
 ==== Properties deprecated in JavaExec task
 
 * The link:{groovyDslPath}/org.gradle.api.tasks.JavaExec.html#org.gradle.api.tasks.JavaExec:main[main] getters and setters
 in link:{groovyDslPath}/org.gradle.api.tasks.JavaExec.html[JavaExec] task have been deprecated.
 Use the link:{groovyDslPath}/org.gradle.api.tasks.JavaExec.html#org.gradle.api.tasks.JavaExec:mainClass[mainClass] property instead.
```

Comment:
Those I had to leave in the upgrade guide because `DefaultJavaExecSpec` and `JavaExecHandleBuilder` which implement the deprecated `JavaExecSpec` methods live in the `core` module while the `JavaExec` task for which the DSL reference exists, lives in `language-java`.